webpack.config.js

```diff
@@ -366,7 +366,7 @@ if (isDev) {
 
 if (isDev || isTest) {
 
-    config.devtool = 'source-map';
+    config.devtool = 'inline-source-map';
```

Comment:
can you check to see how performance of build is in DEV mode?  my worry is that this (without grepping) will produce gargantuan bundle which browser debugger may have trouble with.     Obviously we aren't grepping when we run all tests on CI also, so  it could impact that too.  is there a reason to do inline?

Reply:
Ok, it all seems to work and there are no performance issues on my computer. I do get an error message in the karma browser that I did not observe before. These errors also appear when running the `karma test:watch` command. Tests and debugging seem to work though. 

![Screenshot from 2019-04-03 16-26-04](https://user-images.githubusercontent.com/745885/55486938-7a7c6300-562d-11e9-8575-375df328198a.png)

Reply:
I will investigate other source map options and report back.

Reply:
A new way of using inline-source-map has been implemented. The inline-source-map is now activated via the `--debug` flag passed to karma. This way the inline-source-map is not used when testing on circleCI.